service/src/integrationTest/kotlin/fi/espoo/evaka/vasu/VasuTemplateQueriesTest.kt

```diff
@@ -0,0 +1,71 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.vasu
+
+import fi.espoo.evaka.PureJdbiTest
+import fi.espoo.evaka.resetDatabase
+import fi.espoo.evaka.shared.domain.FiniteDateRange
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import java.time.LocalDate
+
+class VasuTemplateQueriesTest : PureJdbiTest() {
+
+    @AfterEach
+    fun afterEach() {
+        db.transaction { tx ->
+            tx.resetDatabase()
+        }
+    }
+
+    @Test
+    fun `vasu template can be created and updated`() {
+        assertDoesNotThrow {
```

Comment:
I think this function is unnecessary. Test fails by default if it throws.

Reply:
Right, it's unnecessary here. I added them for explicitness in the unit tests and they were left over from test file copy-paste 😄

Reply:
also kotlin.test does not have it :D `assertThrows` is there as `assertFails`, IIRC

Reply:
Refactored.